src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -87,6 +87,13 @@ private async Task<List<IAPIResponse>> ProcessDataTable(string entityType, DataT
             List<IAPIResponse> _apiResponses = [];
             var baseUrl = $"data/ed-fi";
 
+            foreach (var descriptor in dataTable.ExtractDescriptors())
```

Comment:
I think this was mistakenly removed in DMS-259. Bringing it back.

Reply:
Oops 🤦‍♂️